presto-main/src/main/java/com/facebook/presto/cost/LegacyCostCalculator.java

```diff
@@ -1,71 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.cost;
-
-import com.facebook.presto.Session;
-import com.facebook.presto.spi.type.Type;
-import com.facebook.presto.sql.planner.Symbol;
-import com.facebook.presto.sql.planner.plan.PlanNode;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.inject.Inject;
-
-import java.util.HashMap;
-import java.util.Map;
-
-import static com.facebook.presto.util.ImmutableCollectors.toImmutableList;
-
-/**
- * Legacy cost calculator for existing code that uses the old CostCalculator API
- *
- * New code should use the new costCalculator API through the Lookup.getCost() method
- */
-@Deprecated
-public class LegacyCostCalculator
```

Comment:
I don't think you need to in single PR make something to be legacy and remove it. If you are going to remove it, then maybe remove it in first place (squash changes).